apps/docs/src/app/core/component-docs/combobox/examples/combobox-group-example.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-combobox-group-example',
+    templateUrl: './combobox-group-example.component.html'
+})
+export class ComboboxGroupExampleComponent {
+
+    searchTerm = '';
+
+    dropdownValues = [
+        {name: 'Apple', type: 'Fruits'},
+        {name: 'Banana', type: 'Fruits'},
+        {name: 'Pineapple', type: 'Fruits'},
+        {name: 'Strawberry', type: 'Fruits'},
+        {name: 'Broccoli', type: 'Vegetables'},
+        {name: 'Carrot', type: 'Vegetables'},
+        {name: 'Jalapeño', type: 'Vegetables'},
+        {name: 'Spinach', type: 'Vegetables'}
+    ];
+
+    displayFunc(obj: {name: string, price: string}): string {
+        if (obj) {
+            return obj.name;
+        }
+    }
+
+    groupFunc(items: any): {} {
```

Comment:
I think this should be simplified to:
```
return {
    Fruits: items.filter(item => item.type === 'Fruits'),
    Vegetables: items.filter(item => item.type === 'Vegetables')
}
```
Also type adnotations should be corrected.